pkgs/development/python-modules/pennylane-catalyst/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchurl,
+  stdenv,
+}:
+
+buildPythonPackage rec {
+  pname = "pennylane-catalyst";
+  version = "0.13.0";
+  format = "wheel";
+
+  src =
+    fetchurl
```

Comment:
Use `fetchFromGitHub` and build from source. Ask for help if you need it, we're all happy to help.